File: app/src/main/java/com/widyatama/nurseassistant/features/measurement/MeasurementPresenter.java
package com.widyatama.nurseassistant.features.measurement;

import com.widyatama.core.base.BasePresenter;
import com.widyatama.core.util.SchedulerProviderUtil;

import io.reactivex.disposables.CompositeDisposable;

public class MeasurementPresenter<V extends MeasurementViewContracts> extends BasePresenter<V>
        implements MeasurementPresenterContracts<V> {

    public MeasurementPresenter(SchedulerProviderUtil schedulerProvider, CompositeDisposable compositeDisposable){
        super(schedulerProvider, compositeDisposable);
    }
}

File: app/src/main/java/com/widyatama/nurseassistant/features/measurement/MeasurementPresenter.kt
package com.widyatama.nurseassistant.features.measurement

import com.widyatama.core.base.BasePresenter
import com.widyatama.core.util.SchedulerProviderUtil
import io.reactivex.disposables.CompositeDisposable

class MeasurementPresenter<V: MeasurementViewContracts>
constructor(schedulerProvider: SchedulerProviderUtil, compositeDisposable: CompositeDisposable) : BasePresenter<V>(
        schedulerProvider, compositeDisposable), MeasurementPresenterContracts<V> {
}